frontend/src/features/admin-form/responses/queries.ts

```diff
@@ -91,9 +91,10 @@ export const useFormResponses = ({
     () => getFormSubmissionsMetadata(formId, params),
     {
       refetchOnMount: true,
-      refetchOnWindowFocus: false,
-      // Data will never change.
-      staleTime: Infinity,
+      // Consider data stale after 10s, prevent admins from initiating too many
+      // fetches by continuously clicking out and in of the results tab to
+      // refetch data
+      staleTime: 10000,
```

Comment:
setting staletime to `0` should get the behaviour we want: which is to refetch every time the component mounts.

further reading: https://react-query-v3.tanstack.com/guides/initial-query-data#staletime-and-initialdataupdatedat

Reply:
refetching on mount is good because this means users only get the updated information when they renavigate (or refocus) to the page

Reply:
[71edda3](https://github.com/opengovsg/FormSG/pull/4821/commits/71edda30fc6bcecda0ea2ed430b7fc217a8de05f)